sequence_tagging_for_ner/README_en.md

```diff
@@ -0,0 +1,173 @@
+The  program under this directory needs to use the PaddlePaddle v0.10.0 version. If your PaddlePaddle  version is below, update it follow the instructions in the installation [document](http://www.paddlepaddle.org/docs/develop/documentation/en/build_and_install/pip_install_en.html)
```

Comment:
also, there is a confusion when you say "if your paddlepaddle version is below", people may try to look up the version from a paragraph below this one. how about phrase it as:
"The program under this directory depends on PaddlePaddle v0.10.0 or upper.   update it following this instruction if your PaddlePaddle version is lower."